AgentInterfaces/SessionHostsStartInfo.cs

```diff
@@ -86,6 +86,11 @@ public class SessionHostsStartInfo
         /// </summary>
         public CertificateDetail[] GameCertificates { get; set; }
 
+        /// <summary>
+        /// List of secrets we want to deploy on each container
```

Comment:
why on each container? are we planning to support only on container?

Reply:
I copied the same from CertificateDetail class. I want to make it consistent. But your are right. Maybe container is not the right word